CONTRIBUTING

```diff
@@ -260,8 +260,10 @@ Useful SciPy documents
   - The `how to document`_ guidelines
   - NumPy/SciPy `testing guidelines`_
   - `SciPy API`_
+  - The `SciPy 1.0 Roadmap`_
```

Comment:
Might want to drop the 1.0 qualifier. The document seems to evolve into a more general roadmap for the scipy project.